docs/topics/OAuth2.md

```diff
@@ -78,7 +78,13 @@ A URL can be generated that redirects authenticated users to the add-bot flow, b
 https://discordapp.com/api/oauth2/authorize?client_id=157730590492196864&scope=bot&permissions=0
 ```
 
-`client_id` is your _bot_ application's ID and permissions is an integer following the [permissions](#DOCS_PERMISSIONS/bitwise-permission-flags) format.
+(this utilizes the OAuth2 authentiction flow, with a callback URL):
+
+```
+https://discordapp.com/api/oauth2/authorize?client_id=157730590492196864&scope=bot&permissions=0&redirect_uri=https%3A%2F%2Fnicememe.website
+```
+
+`client_id` is your _bot_ application's ID, permissions is an integer following the [permissions](#DOCS_PERMISSIONS/bitwise-permission-flags) format and if supplied the `redirect_uri` will have extra query string parameters `guild_id` and `permissions`. `guild_id` is the guild the _bot_ has joined and `permissions` will be the same sent at the beginning of the OAuth2 authentication flow.
```

Comment:
Let's break this up into more readable sentences:

`client_id` is your bot's application ID. `permissions` is an integer following the [permissions](#DOCS_PERMISSIONS/bitwise-permission-flags) format. Once redirected to your `redirect_uri`, you'll find additional query string parameters: `guild_id`, the guild the bot has joined, and `permissions`, the same integer sent at the start of the flow.